File: nan21.dnet.module.hr/nan21.dnet.module.hr.ui.extjs/src/main/webapp/src/ds/JobTypeLovDs.js
 
   	 
Ext.define("net.nan21.dnet.module.hr.job.ds.model.JobTypeLovDs" , {
	extend: 'Ext.data.Model',
	fields: [  		
		{name:"id",type:"int", useNull:true }, 		
		{name:"clientId",type:"int", useNull:true }, 		
		{name:"name",type:"string" }, 		
		{name:"active",type:"boolean" }
	]
});
	
Ext.define("net.nan21.dnet.module.hr.job.ds.param.JobTypeLovDsParam", {
	extend: 'Ext.data.Model',
	fields: [ 
	]
}); 
 

File: nan21.dnet.module.hr/nan21.dnet.module.hr.presenter/src/main/java/net/nan21/dnet/module/hr/job/ds/model/JobTypeLovDs.java
/*
 * DNet eBusiness Suite
 * Copyright: 2008-2012 Nan21 Electronics SRL. All rights reserved.
 * Use is subject to license terms. 
 */
package net.nan21.dnet.module.hr.job.ds.model;

import net.nan21.dnet.core.api.annotation.Ds;
import net.nan21.dnet.core.api.annotation.SortField;
import net.nan21.dnet.core.presenter.model.base.AbstractTypeLov;
import net.nan21.dnet.module.hr.job.domain.entity.JobType;

@Ds(entity = JobType.class, jpqlWhere = " e.active = true ", sort = { @SortField(field = JobTypeLovDs.fNAME) })
public class JobTypeLovDs extends AbstractTypeLov<JobType> {

    public JobTypeLovDs() {
        super();
    }

    public JobTypeLovDs(JobType e) {
        super(e);
    }

}
